routes/web.php

```diff
@@ -13,6 +15,9 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::redirect('/', '/login');
+
+Route::get('/register', [RegisterController::class, 'index'])->name('user.register');
+Route::post('/register', [RegisterController::class, 'register']);
+
+Route::view('/login', 'auth.login')->name('user.login');
```

Comment:
rather can returning view from routes. define a `LoginController` and return the login view  from controller method. This way you can resuse the controller for handling login action as well  later on